pkgs/applications/terminal-emulators/kitty-themes/default.nix

```diff
@@ -0,0 +1,28 @@
+{
+  fetchFromGitHub, lib, stdenv
+}:
+
+stdenv.mkDerivation rec {
+  pname = "kitty-themes";
+  version = "2021-01-04";
```

Comment:
This date should reflect the date when the commit below was made. In this case it should be 2022-01-01.